applications/StructuralMechanicsApplication/custom_constitutive/timoshenko_beam_elastic_constitutive_law.cpp

```diff
@@ -138,11 +138,15 @@ void TimoshenkoBeamElasticConstitutiveLaw::CalculateMaterialResponseCauchy(Const
 
         AddInitialStressVectorContribution(r_generalized_stress_vector);
 
+        if (r_material_properties.Has(BEAM_PRESTRESS_PK2)) {
+            r_generalized_stress_vector[0] += r_material_properties[BEAM_PRESTRESS_PK2][0] * A;
+        }
```

Comment:
This implementation seems to be incorrect. The beam is formulated in resultant axial forces, bending moments and shear forces. This implementation only takes the axial force and interprets it as a stress. For the beams, desired is all components as resultant forces/moments (unlike for trusses, where we only take into account the axial term, which there really is a stress). 
```suggestion
        if (r_material_properties.Has(BEAM_PRESTRESS_PK2)) {
            r_generalized_stress_vector += r_material_properties[BEAM_PRESTRESS_PK2];
        }
```
This fixes the failing test